index.js

```diff
@@ -121,25 +122,15 @@ class Realm {
   }
 
   define(name, attributes, opts = {}, descriptors = {}) {
-    const { Bone } = this;
-    const Model = class extends Bone {
-      // export Model: instance.Model.name
-      get Model() {
-        return Model;
-      }
-      // export Model: class.Model.name
-      static get Model() {
-        return Model;
-      }
```

Comment:
`prototype.Model` is handled in sequelize adapter specifically

`class.Model` isn't necessary because `class.name` is guaranteed in egg-orm when subclassing.